docs/concepts/what-is-a-schedule.md

```diff
@@ -85,15 +85,21 @@ The time zone definition is loaded on the Temporal Server Worker Service from ei
 For more operational control, embed the contents of the time zone database file in the Schedule Spec itself.
 (Note: this isn't currently exposed in tctl or the web UI.)
 
-### Pausing
+### Pause
 
 A Schedule can be Paused.
 When a Schedule is Paused, the Spec has no effect.
 However, you can still force manual actions by using the [tctl schedule trigger](/tctl/schedule/trigger) command.
 
 To assist communication among developers and operators, a “notes” field can be updated on pause or resume to store an explanation for the current state.
 
-### Limiting number of Actions
+### Backfill
+
+A Schedule can be Backfilled.
+When a Schedule is Backfilled, all the Actions that would have been taken over a specified time period are taken now (in parallel if the `AllowAll` [Overlap Policy](#overlap-policy) is used; sequentially if `BufferAll` is used).
+You might use this to fill in runs from a time period when the Schedule was paused due to an external condition that's now resolved, or a period before the Schedule was created.
+
+### Limit number of Actions
```

Comment:
These section headers read as nouns? So
```suggestion
### Limited number of Actions
```
?

Reply:
Some are verbs. I'm matching the verb tense used elsewhere in docs with Pause, Backfill, and Limit